conda-forge.yml

```diff
@@ -1,3 +1,3 @@
 build_platform: {osx_arm64: osx_64, linux_ppc64le: linux_64, linux_aarch64: linux_64}
 conda_forge_output_validation: true
-test_on_native_only: true
+test: native
```

Comment:
Please revert this. This is only masking an issue in the build. The error is not an emulation issue.